testCourse/questions/subfolder/nestedQuestion/info.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I think the nested question is MIA from an assessment. Could you add it real quick?

Reply:
oh no, that's somehow completely broken the whole assessment now :cry:

Reply:
never mind, brain fart